patches/server/1037-General-ItemMeta-fixes.patch

```diff
@@ -906,6 +908,15 @@ index 705d7365a7bab9fe29b90a175040e6a1623e21a7..8ccb8d71ee489891e8d9128a5520675d
          this.lore = lore != null ? io.papermc.paper.adventure.PaperAdventure.asVanilla(lore) : null;
      }
      // Paper end
+@@ -1094,7 +1099,7 @@ class CraftMetaItem implements ItemMeta, Damageable, Repairable, BlockDataMeta {
+     @Override
+     public void removeEnchantments() {
+         if (this.hasEnchants()) {
+-            this.enchantments.clear();
++            this.enchantments = null;
```

Comment:
needs a paper comment, `// Paper - Correctly clear enchantments`

Reply:
added comment

Reply:
Should be on the same line, after the ;

Reply:
fixed ._.